trade-finance-manager-ui/server/controllers/case/cancellation/effective-from-date.controller.ts

```diff
@@ -51,6 +52,7 @@ export const getEffectiveFromDate = async (req: GetEffectiveFromDateRequest, res
       day,
       month,
       year,
+      backUrl: status === 'change' ? `/case/${_id}/cancellation/check-details` : `/case/${_id}/cancellation/bank-request-date`,
```

Comment:
FWIW could add a function to abstract this logic
```suggestion
      backUrl: getPreviousPage(_id, status),
```

Reply:
Extracted, thanks!